src/Stratis.Bitcoin.Features.BlockStore.Tests/PruneBlockStoreServiceTests.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System.Threading.Tasks;
```

Comment:
Should this whole class go in light wallet tests?

Reply:
Hi @dangershony yes, `PruneBlockRepository` and `PruneBlockStoreService` will now be in the blockstore feature, so the tests should be here as well.

They are both block store features/components and is not specific to the light wallet per se 👍